demo/demo_2d_unbounded.py

```diff
@@ -9,8 +9,8 @@
 ti.init(arch=ti.cuda)  # Try to run on GPU
 
 gui = ti.GUI("Taichi MLS-MPM", res=512, background_color=0x112F41)
-
-mpm = MPMSolver(res=(128, 128), unbounded=True)
+# set padding bigger to avoid particles going out of bound
+mpm = MPMSolver(res=(128, 128), unbounded=True, padding=16)
```

Comment:
```suggestion
mpm = MPMSolver(res=(128, 128), unbounded=True)
```

Let's set default `padding` to 16 then.